### PR TITLE
Correct type annotations and docstrings in `XYData` and `XYSession`

### PR DESCRIPTION
**Pull request by Diogo-Rossi**
Small pull request making compatible methods of `XYData` and `XYSession` classes.

Solves #1189, from discussion https://github.com/haiiliin/abqpy/discussions/1188

### CODE REVIEW

**Comment by haiiliin** [comment]
Nice job!

File: src/abaqus/XY/XYSession.py [hunk 1]
```diff
@@ -471,13 +472,18 @@ def XYDataFromFile(
             by spaces, tabs, or commas. The default value is 2.
         skipFrequency
             An Int specifying how often data rows will be skipped. A **skipFrequency** of 1 means skip
-            every other row. The first row is always read. Possible values are **skipFrequency** ≥ 0.
+            every other row. The first row is always read. Possible values are **skipFrequency** ≥≥ 0.
```
Comment (haiiliin) [inline src/abaqus/XY/XYSession.py:475]: Remove the extra ≥

File: src/abaqus/XY/XYSession.py [hunk 2]
```diff
@@ -537,7 +543,7 @@ def XYDataFromHistory(
         skipFrequency
             An Int specifying how often data frames will be skipped. If **skipFrequency** = 1, Abaqus
             will skip every other frame. The first frame is always read. Possible values are
-            **skipFrequency** ≥ 0. The default value is 0 (data are read from every frame).
+            **skipFrequency** ≥≥ 0. The default value is 0 (data are read from every frame).
```
Comment (haiiliin) [inline src/abaqus/XY/XYSession.py:546]: Remove the extra ≥

File: src/abaqus/XY/XYSession.py [hunk 3]
```diff
@@ -596,20 +607,21 @@ def xyDataListFromField(
             WHOLE_PART_INSTANCE, and WHOLE_REGION.Refinement: A tuple specifying the refinement. If
             the refinement tuple is omitted, data are written for all components and invariants (if
             applicable). This element is required if the location dictionary (the following element
-            in the tuple) is included. The refinement tuple contains the following Type: A
+            in the tuple) is included. The refinement tuple contains the following:Type: A
```
Comment (haiiliin) [inline src/abaqus/XY/XYSession.py:610]: change : to a space

File: src/abaqus/XY/XYSession.py [hunk 4]
```diff
@@ -706,7 +723,104 @@ def XYDataFromFreeBody(
...
+            A sequence of Strings specifying element sets or a String specifying a single element
+            set.
+        elementLabels
+            A sequence of expressions specifying element labels per part instance in the model. Each
+            part instance element expression is a sequence of a String specifying the part instance
+            name and a sequence of element expressions; for example,
+            `(('partInstance1',(1,'7','3:15;3'),), ('partInstance2','8'),))`. The element
+            expressions can be any of the following:An Int specifying a single element label; for
+            example, `1`.A String specifying a single element label; for example, `'7'`.A String
+            specifying a sequence of element labels; for example, `'3:5'` and `'3:15:3'`.
```
Comment (haiiliin) [inline src/abaqus/XY/XYSession.py:796]: Can you reformat the code blocks like it in other methods? And also in the XYData file

**Comment by haiiliin** [comment]
Rebase the branch to adopt #1191.

File: src/abaqus/XY/XYSession.py [hunk 5]
```diff
@@ -771,9 +793,14 @@ def XYDataFromPath(
...
             A Boolean specifying whether to remove duplicate XY values from the final result. The
             default value is True.
+
+            .. versionadded:: 2018
+                The `removeDuplicateXYPairs` argument was added.
         includeAllElements
             A Boolean specifying whether to include elements which do not lie in the direction of
             the path. The default value is False.
+            .. versionadded:: 2018
+                The `includeAllElements` argument was added.
```
[the remark this answers is not in the document]
Comment (haiiliin) [reply src/abaqus/XY/XYSession.py:895]: Pages build failed because of this.

File: src/abaqus/XY/XYSession.py (hunk 4 above)
Comment (Diogo-Rossi) [reply src/abaqus/XY/XYSession.py]: Sorry, I was working on that, but did not pushed.

**Pull request merged**